pom.xml

```diff
@@ -32,17 +32,17 @@
   </modules>
 
   <properties>
-    <cobertura.version>2.0.3</cobertura.version>
-    <cobertura.maven.plugin.version>2.6</cobertura.maven.plugin.version>
+    <cobertura.version>2.1.1</cobertura.version>
+    <cobertura.maven.plugin.version>2.7</cobertura.maven.plugin.version>
     <maven.release.plugin.version>2.5.3</maven.release.plugin.version>
     <spring-platform.version>Brussels-SR5</spring-platform.version>
     <hadoop.version>2.7.2</hadoop.version>
-    <hive.version>2.3.0</hive.version>
-    <dropwizard.version>1.0.5</dropwizard.version>
+    <hive.version>2.3.3</hive.version>
```

Comment:
Call this one out in the changelog as it does affect usage outside of the internals of this project - say something like "Upgraded default Hive version from 2.3.0 to 2.3.3"